src/shared/components/palette/base-field/abstract-form-field.component.ts

```diff
@@ -65,10 +65,14 @@ export abstract class AbstractFormFieldComponent {
 
   private addControlToFormGroup(control: AbstractControl, parent: FormGroup, replace: boolean): AbstractControl {
     if (replace) {
-      parent.setControl(this.caseField.id, control);
+      if (control['controls'] && control['controls']['Id'] && control['controls']['Id'].value === 'LinkedCase') {
+        parent.setControl('CaseReference', control['controls']['CaseReference']);
+      } else {
+        parent.setControl(this.caseField.id, control);
+      }
     } else {
       parent.addControl(this.caseField.id, control);
-    }
+    };
```

Comment:
Semi colon after a closing if clause? It is probably not breaking anything but let's not do that as it isn't standard practise.

Reply:
i will remove this now.